pavelib/prereqs.py

```diff
@@ -185,6 +185,7 @@ def install_node_prereqs():
     "django-storages",
     "django-oauth2-provider",       # Because now it's called edx-django-oauth2-provider.
     "edx-oauth2-provider",          # Because it moved from github to pypi
+    "pytz",                         # Because Uruguay doesn't celebrate DST, but might again next year
```

Comment:
If we're pinning an exact version, we don't need to add it to the reinstall list.